### PR TITLE
feat: zwave-js@10.7.0, heal network options, endpoints labels (#2941

### PR DESCRIPTION
**Pull request by robertsLando**
Removed `endpointIndezes` in favor of `endpoint` property from ZUINode

### CODE REVIEW

File: src/views/ControlPanel.vue [hunk 1]
```diff
@@ -413,6 +409,28 @@ export default {
...
+							'Healing network causes a lot of traffic, can take minutes up to hours and users have to expect degraded performance while it is going on',
+							'info',
+							{
+								confirmText: 'Heal',
+								inputs: [
+									{
+										type: 'checkbox',
+										label: 'Include sleeping nodes',
+										key: 'includeSleeping',
+										value: false,
```
Comment (AlCalzone) [inline src/views/ControlPanel.vue:425]: Default is `true` in the driver for compatibility reasons. I guess default `false` here is okay.

File: src/components/nodes-table/AssociationGroups.vue [hunk 2]
```diff
@@ -128,6 +134,15 @@ export default {
 			const node = this.nodes[this.nodesMap.get(nodeId)]
 			return node ? node._name : 'NodeID_' + nodeId
 		},
+		getEndpointLabel(nodeId, endpoint) {
+			const node = this.nodes[this.nodesMap.get(nodeId)]
+			if (node) {
+				const ep = node.endpoints.find((e) => e.index === endpoint)
+				return ep?.label ? ep.label : 'Endpoint ' + endpoint
```
Comment (AlCalzone) [inline src/components/nodes-table/AssociationGroups.vue:141]:
```suggestion
				return ep?.label || 'Endpoint ' + endpoint
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: src/components/dialogs/DialogAssociation.vue [hunk 3]
```diff
@@ -188,9 +188,12 @@ export default {
 				endpoints.unshift({ text: 'No Endpoint', value: null })
 			}
 
-			if (node && node.endpointIndizes) {
-				for (const i of node.endpointIndizes) {
-					endpoints.push({ text: 'Endpoint ' + i, value: i })
+			if (node && node.endpoints) {
+				for (const endpoint of node.endpoints) {
+					endpoints.push({
+						text: endpoint.label || 'Endpoint ' + endpoint.index,
```
[the remark this answers is not in the document]
Comment (AlCalzone) [reply src/components/dialogs/DialogAssociation.vue:194]: this is the actual change request, the other comments are just information/hints

File: src/components/nodes-table/AssociationGroups.vue (hunk 2 above)
Comment (robertsLando) [reply src/components/nodes-table/AssociationGroups.vue]: Yeah, didn't noticed this, even better:

```
if (node) {
				const ep = node.endpoints.find((e) => e.index === endpoint)
				if (ep) {
					return ep.label
				}
			}

			return 'Endpoint ' + endpoint
```

File: src/components/dialogs/DialogAssociation.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (robertsLando) [reply src/components/dialogs/DialogAssociation.vue:194]: I think we could do this on backend side directly, I mean set the label to that on root endoint and if not set

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**